### DISCUSSION TITLE
Add widgets

### DISCUSSION THREAD

**Discussion by ozoramore** [jocmp/capyreader#197] [Feature Requests]
Are you planning to add widget to the app?
I use widgets a lot, so it would be really helpful.

**Comment by jocmp** [comment]
@mori-oh I hadn't thought about it, but widgets are neat. What kind of widget would you want to see, or is there an RSS widget you already use that you'd want to see here?

**Reply by ozoramore** [reply]
Thanks for considering.
I just want to know the headline. 
So the widget I want is similar to the message list widget in "K-9 Mail" or "gmail".
